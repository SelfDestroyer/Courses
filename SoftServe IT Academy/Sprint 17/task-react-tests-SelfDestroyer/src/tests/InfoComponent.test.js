import TestRenderer from "react-test-renderer";
import Info from "../components/Info";

describe("Test comnonent Info", () => {
  let componentInfo;

  beforeEach(() => {
    componentInfo = TestRenderer.create(<Info user="yurkovskiy" />);
  });

  it("Render corectly", () => {
    expect(componentInfo.toJSON()).toMatchSnapshot();
  });

  it("Test props", () => {
    const testInstance = componentInfo.toTree();
    expect(testInstance.props.user).toEqual("yurkovskiy");
  });
});
