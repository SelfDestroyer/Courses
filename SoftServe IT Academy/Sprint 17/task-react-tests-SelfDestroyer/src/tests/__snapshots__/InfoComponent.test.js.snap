// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test comnonent Info Render corectly 1`] = `
<div>
  <h3>
    GitHub User Info
  </h3>
  <ul>
    <li>
      error
      : 
      request error
    </li>
  </ul>
</div>
`;
